adoc/deployment-azure.adoc

```diff
@@ -0,0 +1,217 @@
+to be provided later to Azure, hence it's strongly recommended to create
+a dedicate key pair.
+
+Once the whole infrastructure is created you can connect into any node of the
+cluster by doing the following steps:
+
+  * Log into Azure portal
+  * Choose one of the nodes of the cluster
+  * Click "connect" and select "bastion" as option
+  * Enter all the required fields
+
+Once this is done a new browser tab will be open with a shell session running
+inside of the desired node.
+
+It's recommended to use Chrome or Chromium during this process.
+
+You can ssh into the first master node of the cluster of drive the creation of
+a SUSE cluster from there. Once this is done you can download a kubeconfig
+file and operate the cluster without having to go through the bastion host.
+
+Caveats of Azure Bastion:
+
+  * As of June 2020, the link:https://docs.microsoft.com/en-us/azure/bastion/bastion-overview#regions[Azure Bastion service] is not available in all Azure regions.
+  * By design it's not possible to leverage the bastion host without using the
+    ssh session embedded into the browser. This makes impossible to use tools like
+    `sftp` or `scp`.
+  * You have to rely on copy and paste to share data (like the `admin.conf` file
+    generated by skuba) between the remote nodes and your local system.
+    You can "rely" on `cat`, `base64` and a lot of copy and paste...
+  * `skuba` requires a private ssh key to connect to all the nodes of the cluster.
+    You have to upload the private key you specified at cluster creation
+    or create a new one inside of the first master node and copy that
+    around the cluster.
+
+=== Virtual Network Peering Support
+
+It is possible to join existing network to the cluster.  It can be setup by adding a list of network id to `peer_virutal_network_id`.
+
+=== Enable Multiple Zones
+
+It is possible to enable multiple zone.  It can be set `enable_zone` to `true` and master/worker node will distribute sequentially based on zones defined in `azure_availability_zones`. 
+
+  * As of June 2020, the link:https://docs.microsoft.com/en-us/azure/availability-zones/az-region[Azure Availability Zones] is not available in all Azure regions.
+
+=== Deploying the Infrastructure
+
+On the management machine, find the {tf} template files for AZURE in
+`/usr/share/caasp/terraform/azure`. These files have been installed as part of
+the management pattern (`sudo zypper in -t pattern SUSE-CaaSP-Management`).
+
+Copy this folder to a location of your choice as the files need adjustment.
+
+----
+mkdir -p ~/caasp/deployment/
+cp -r /usr/share/caasp/terraform/azure/ ~/caasp/deployment/
+cd ~/caasp/deployment/azure/
+----
+
+Once the files are copied, rename the `terraform.tfvars.example` file to
+`terraform.tfvars`:
+
+----
+cp terraform.tfvars.example terraform.tfvars
+----
+
+Edit the `terraform.tfvars` file and add/modify the following variables:
+
+include::deployment-terraform-example.adoc[tags=tf_azure]
```

Comment:
Missing terraform config examples.

Reply:
I think you haven't updated the https://github.com/SUSE/doc-caasp/blob/master/adoc/deployment-terraform-example.adoc

Reply:
added